src/fsck/fsck.c

```diff
@@ -365,7 +366,10 @@ static int run(int argc, char *argv[]) {
                 } else
                         dash_c[0] = 0;
 
-                cmdline[i++] = "/sbin/fsck";
+                r = find_executable("fsck", &fsck_path);
+                if (r < 0)
+                        return r;
```

Comment:
`_exit`?

Reply:
You mean this should be a `return log_error_errno`?

Reply:
This is child process, and thus should `_exit` directly to avoid running cleanup functions or such.

Please log this error and do `_exit(FSCK_OPERATIONAL_ERROR);`